src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/TicketTabNav_PurchaseTabScreen.tsx

```diff
@@ -33,32 +35,76 @@ export const TicketTabNav_PurchaseTabScreen = ({navigation}: Props) => {
   const showTicketAssistant = useTicketingAssistantEnabled();
   const analytics = useAnalytics();
 
+  const {remoteTokens} = useMobileTokenContextState();
+  const inspectableToken = findInspectable(remoteTokens);
+  const isMobileTokenEnabled = isMobileToken(inspectableToken);
```

Comment:
I think this variable name can be renamed, as `mobileTokenEnabled` is a phrase used throughout the app meaning if mobile token is enabled in RemoteConfig or not. Maybe something like `HasInspectableMobileToken` would do?

Reply:
Fixed